pipeline.go

```diff
@@ -456,6 +485,11 @@ func (p Pipeline) PullImages(force bool) error {
 	if Verbose {
 		pipelineLogger.Printf("Total time spent pulling images: %s", totalElapsedTime)
 	}
+	select {
+	case <-abort:
+		return fmt.Errorf("error during image pulling!")
```

Comment:
This sounds a bit too muh like "hair pulling" to me. Maybe "An error occured while pulling images"